### PR TITLE
clang check misc-uniqueptr-reset-release

### PR DESCRIPTION
**Pull request by davidlange6**
for comment (and likely not mergeable - this is what clang-tidy check misc-uniqueptr-reset-release does

### CODE REVIEW

**Comment by davidlange6** [comment]
please test

File: L1TriggerConfig/GMTConfigProducers/src/L1MuGMTParametersOnlineProducer.cc [hunk 1]
```diff
@@ -127,7 +127,7 @@ void L1MuGMTParametersOnlineProducer::checkCMSSWVersion(const coral::AttributeLi
   /* PROJECT_VERSION is passed as a -D #define from scramv1 (eg CMSSW_2_1_0) */
   if(versionString != PROJECT_VERSION) { 
     std::string errMsg = "CMSSW version mismatch: Configuration requires " + 
-      versionString + ", but this is " PROJECT_VERSION "!";
+      versionString + ", but this is "; PROJECT_VERSION "!";
```
Comment (Dr15Jones) [inline L1TriggerConfig/GMTConfigProducers/src/L1MuGMTParametersOnlineProducer.cc]: Bad

**Comment by Dr15Jones** [comment]
+1
Framework looks good

**Comment by slava77** [comment]
+1

https://stackoverflow.com/questions/13860219/what-are-the-differences-between-stdmove-and-unique-ptrreset
is apparently more descriptive than the docs
https://clang.llvm.org/extra/clang-tidy/checks/misc-uniqueptr-reset-release.html

**Pull request closed without merging**